### ISSUE TITLE
Australia not showing testing data

### ISSUE THREAD

**Issue by benrada** [CSSEGISandData/COVID-19#1900]
Testing data sits at zero for Australia.

**Comment by Dagude14** [comment]
where do you see the testing data?

**Comment by appastair** [comment]
I don't believe this dataset tracks the number of tests conducted nor do the total number of tests get published by health departments regularly. :woman_shrugging:

**Comment by MichaelAlexi** [comment]
Desperately looking for testing data sets to make a time series. Also looking for South Korea and Saudi Arabia. Can anyone send me a CSV link? 

Alternatively has anyone got scraped pages saved? 

Aussie testing Data is summaried daily in a:
PDF https://www.health.gov.au/resources/publications/coronavirus-covid-19-at-a-glance
HTML: https://www.health.gov.au/news/health-alerts/novel-coronavirus-2019-ncov-health-alert/coronavirus-covid-19-current-situation-and-case-numbers
7 April 2020: "Of the 5,844 confirmed cases in Australia, 42 have died from COVID-19. More than 304,000 tests have been conducted across Australia."
